### DISCUSSION TITLE
Overlay BigWig tracks?

### DISCUSSION THREAD

**Discussion by elhossary** [GMOD/jbrowse-components#2865] [General] [answered]
Hi,
I am using MultiBigWig plugin with JBrowse v1 to merge multiple BigWig tracks into one.
How can I do this in JBrowse 2?

Thanks

**Comment by cmdcolin** [comment, marked as answer]
Glad to hear interest in this! We do not have a multibigwig equivalent in JBrowse 2 but I agree, it would be really useful!

**Reply by elhossary** [reply]
Thanks for the quick reply, please add it to the features shopping list :)

**Reply by cmdcolin** [reply]
hello @elhossary some development on a "multibigwig" track has started here. if you have any interest in testing it out and have any questions or feedback, the current branch is here https://github.com/GMOD/jbrowse-components/pull/3043 

it has the ability to "add multiple bigwig tracks to a selection" in the track selector, which creates a shopping cart, and then you can create a multibigwig from items in the shopping cart
